src/tests/functional/plugin/shared/src/low_precision_transformations/mat_mul_transformation.cpp

```diff
@@ -21,10 +22,9 @@ namespace LayerTestsDefinitions {
 
 std::string MatMulTransformation::getTestCaseName(const testing::TestParamInfo<MatMulTransformationParams>& obj) {
     ngraph::element::Type precision;
-    ngraph::PartialShape inputShape;
```

Comment:
Why did you remove input shape for these tests?
Is it related with these changes? `ngraph::PartialShape` can be changed to `ov::PartialShape`